testhelper/integrationtesthelper.go

```diff
@@ -116,19 +116,14 @@ func testCartographerPosition(t *testing.T, svc slam.Service, useIMU bool, expec
 
 // Checks the cartographer map and confirms there at least 100 map points.
 func testCartographerMap(t *testing.T, svc slam.Service, localizationMode bool) {
-	timestamp1, err := svc.LatestMapInfo(context.Background())
+	prop, err := svc.Properties(context.Background())
```

Comment:
[nit] Clearer to read
```suggestion
	props, err := svc.Properties(context.Background())
```